packit/base_git.py

```diff
@@ -359,12 +359,9 @@ def set_specfile_content(self, specfile: Specfile, version: str, comment: str):
         :param version: version to set in self.specfile
         :param comment: new comment for the version in %changelog
         """
-        this_changelog = self.specfile.spec_content.section("%changelog")
-        this_version = self.specfile.get_version()
+        provided_changelog = specfile.spec_content.section("%changelog")
         self.specfile.spec_content.sections[:] = specfile.spec_content.sections[:]
-        self.specfile.save()
-        self.specfile.spec_content.replace_section("%changelog", this_changelog)
-        self.specfile.set_version(this_version)
+        self.specfile.spec_content.replace_section("%changelog", provided_changelog)
         self.specfile.save()
         self.specfile.set_spec_version(version=version, changelog_entry=comment)
```

Comment:
I am just wondering if we don't want to call `save` also at the end. Or is it run internally in rebase-helper?

Reply:
I don't know to be honest, will add it.